File: myPackage.dart
class Developer {
  String? name;
  List<String?> languages;
  Developer(this.name, this.languages);
  void addLang(lang) {
    languages.add(lang);
  }
}

class Translator extends Developer {
  Translator(String? name, List<String?> languages) : super(name, languages);
  @override
  void addLang(lang) {
    if (this.languages.length >= 3) {
      this.languages.removeAt(0);
      super.addLang(lang);
    }
  }
}

File: myPackage.py


class Developer:
    def __init__(self,languages,name="Ahmed"):
        self.languages = languages
        self.name = name

    def addLang(self,lang):
        self.languages.append(lang)
        
    
class Translator(Developer):
    def __init__(self, languages, name):
        super().__init__(languages, name=name)

    def addLang(self, lang):
        if len(self.languages) >=3:
            self.languages.remove(self.languages[0])
            return super().addLang(lang)
